### ISSUE TITLE
Fedora iot-bootable-container requires a bootc/install config

### ISSUE THREAD

**Issue by achilleas-k** [osbuild/images#623]
The new bootc-image-builder now uses `bootc install to-filesystem`, which requires the container define a config for the tool to read.
```
ERROR Installing to filesystem: Loading configuration: No bootc/install config found; this operating system must define a default configuration to be installable
Traceback (most recent call last):
  File "/run/osbuild/bin/org.osbuild.bootc.install-to-filesystem", line 53, in <module>
    r = main(args["options"], args["inputs"], args["paths"])
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/run/osbuild/bin/org.osbuild.bootc.install-to-filesystem", line 48, in main
    subprocess.run(pargs, env=env, check=True)
  File "/usr/lib64/python3.12/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['bootc', 'install', 'to-filesystem', '--source-imgref', 'containers-storage:[overlay@/run/osbuild/containers/storage+/run/containers/storage]401ce5fe69f79274dff5db0acc95f23859b593d1f1187a46961bc19b40bdda74', '--skip-fetch-check', '--generic-image', '--karg', 'rw', '--karg', 'console=tty0', '--karg', 'console=ttyS0', '--target-imgref', 'registry.gitlab.com/redhat/services/products/image-builder/ci/images/iot-bootable-container:fedora-39-aarch64-4b3675c1ce01f900bfcadaa5d4cf9a021c063b024ad728b0631f64f048437882', '/run/osbuild/mounts']' returned non-zero exit status 1.
```

For example, the quay.io/centos-bootc/fedora-bootc:eln container contains the file `/usr/lib/bootc/install/20-rhel.toml` with the following content:

```
root-fs-type = "xfs"
```

Refs:
- Build failures from #618 
    - https://gitlab.com/redhat/services/products/image-builder/ci/images/-/jobs/6674381430
- Config docs: https://github.com/containers/bootc/blob/36e70dae9bd3bac91f24c68f1661e279c0ee0ff2/docs/src/man-md/bootc-install-config.md

**Comment by cgwalters** [comment]
This will be fixed by https://github.com/containers/bootc/pull/480

**Comment by achilleas-k** [comment]
Thanks.  Still, it'd be nice to be able to write a config for the bootable containers we build.

**Comment by achilleas-k** [comment]
Fixed by #636
